sklearn/linear_model/base.py

```diff
@@ -32,7 +32,8 @@
 from ..utils.extmath import safe_sparse_dot
 from ..utils.sparsefuncs import mean_variance_axis, inplace_column_scale
 from ..utils.fixes import sparse_lsqr
-from ..utils.seq_dataset import ArrayDataset, CSRDataset
+from ..utils.seq_dataset import ArrayDataset32, CSRDataset32, ArrayDataset, \
```

Comment:
We prefer to avoid the line continuation with a backslash and rather start a new line, repeating the "from ...utils"